critiquebrainz/frontend/external/musicbrainz_db/serialize.py

```diff
@@ -83,19 +83,56 @@ def to_dict_places(place, includes={}):
     return data
 
 
+def to_dict_artist_credit_names(artist_credit_name):
+    data = {
+        'name': artist_credit_name.name,
+        'artist': to_dict_artists(artist_credit_name.artist),
+    }
+    if artist_credit_name.join_phrase:
+        data['join_phrase'] = artist_credit_name.join_phrase
+    return data
+
+
 def to_dict_release_groups(release_group, includes={}):
```

Comment:
Sorry, read the backlogs later (on irc) about the script in `musicbrainz-server` (that way might be better for testing different type of entities)